### PR TITLE
Upgraded Lerna to 3

### LINKED ISSUE #441: Upgrade Lerna
Unfortunately, Lerna 2.11 is not compatible with latest Git:

```
(master d0dc448) p@m /workspace/ckeditor5-dev> lerna publish
lerna info version 2.11.0
lerna info versioning independent
lerna info Checking for updated packages...
lerna info Comparing with @ckeditor/ckeditor5-dev-docs@10.0.1.
lerna ERR! initialize caught error
lerna ERR! Error: fatal: empty string is not a valid pathspec. please use . instead if you meant to match all paths
lerna ERR!
lerna ERR!     at Function.module.exports.sync (/usr/local/lib/node_modules/lerna/node_modules/execa/index.js:303:26)
lerna ERR!     at Function.execSync (/usr/local/lib/node_modules/lerna/lib/ChildProcessUtilities.js:59:30)
lerna ERR!     at Function.diffSinceIn (/usr/local/lib/node_modules/lerna/lib/GitUtilities.js:183:50)
lerna ERR!     at UpdatedPackagesCollector.hasDiffSinceThatIsntIgnored (/usr/local/lib/node_modules/lerna/lib/UpdatedPackagesCollector.js:254:41)
lerna ERR!     at /usr/local/lib/node_modules/lerna/lib/UpdatedPackagesCollector.js:134:24
lerna ERR!     at Array.filter (<anonymous>)
lerna ERR!     at UpdatedPackagesCollector.collectUpdatedPackages (/usr/local/lib/node_modules/lerna/lib/UpdatedPackagesCollector.js:130:23)
lerna ERR!     at UpdatedPackagesCollector.getUpdates (/usr/local/lib/node_modules/lerna/lib/UpdatedPackagesCollector.js:88:35)
lerna ERR!     at PublishCommand.initialize (/usr/local/lib/node_modules/lerna/lib/commands/PublishCommand.js:275:67)
{ Error: fatal: empty string is not a valid pathspec. please use . instead if you meant to match all paths

    at Function.module.exports.sync (/usr/local/lib/node_modules/lerna/node_modules/execa/index.js:303:26)
    at Function.execSync (/usr/local/lib/node_modules/lerna/lib/ChildProcessUtilities.js:59:30)
    at Function.diffSinceIn (/usr/local/lib/node_modules/lerna/lib/GitUtilities.js:183:50)
    at UpdatedPackagesCollector.hasDiffSinceThatIsntIgnored (/usr/local/lib/node_modules/lerna/lib/UpdatedPackagesCollector.js:254:41)
    at /usr/local/lib/node_modules/lerna/lib/UpdatedPackagesCollector.js:134:24
    at Array.filter (<anonymous>)
    at UpdatedPackagesCollector.collectUpdatedPackages (/usr/local/lib/node_modules/lerna/lib/UpdatedPackagesCollector.js:130:23)
    at UpdatedPackagesCollector.getUpdates (/usr/local/lib/node_modules/lerna/lib/UpdatedPackagesCollector.js:88:35)
    at PublishCommand.initialize (/usr/local/lib/node_modules/lerna/lib/commands/PublishCommand.js:275:67)
    at PublishCommand._attempt (/usr/local/lib/node_modules/lerna/lib/Command.js:377:21) exitCode: 1 }
```

I updated git recently and this appeared. 

We need to upgrade Lerna to version 3 here, which means that we need to stop using it in http://github.com/ckeditor/ckeditor5 first (https://github.com/ckeditor/ckeditor5/issues/1214).

When we'll be upgrading to Lerna 3 here we should remove `.` from:

```json
  "packages": [
    "packages/*",
    "."
  ],
```

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Upgraded Lerna to 3.x due to an error that blocks publishing a new version of packages. Closes #441.

### CODE REVIEW

**Comment by pomek** [comment]
PS 1: Don't forget calling lerna from `node_modules` (`./node_modules/.bin/lerna [command]`).

PS 2: Remember about re-installing deps :D

**Comment by Reinmar** [comment]
What about https://github.com/ckeditor/ckeditor5-dev/blob/master/scripts/changelog.js#L10? Do we want to use installed package for that? Or maybe the one from `packages/...`? In the first case we'll need to remember to bump its version manually in `package.json` after releasing it. But perhaps it's good?

**Comment by pomek** [comment]
IMO would be good to go with importing `packages/**/*.js` instead of installing it from npm.

**Comment by Reinmar** [comment]
Good, let's do that :)

**Comment by pomek** [comment]
Done.

File: lerna.json [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
   "packages": [
-    "packages/*",
-    "."
+    "packages/*"
   ],
   "hoist": true,
   "nohoist": "guppy-pre-commit",
   "version": "independent",
-  "commands": {
+  "command": {
```
Comment (Reinmar) [inline lerna.json]: Is bootstrap's concurrency still configurable? I think that this option doesn't exist any more.

**Comment by Reinmar** [comment]
Also, there are warnings about version mismatch.

image

File: lerna.json (hunk 1 above)
Comment (pomek) [reply lerna.json]: image

**Pull request merged**